3.0/schema.json

```diff
@@ -0,0 +1 @@
+schema
```

Comment:
This file is currently invalid json. If the intent is to have a json schema, use `{}` for a schema "which constrains nothing, allows anything, and describes nothing"